test/unit/settlement/process-settlement.test.js

```diff
@@ -39,7 +53,7 @@ describe('process settlement', () => {
   })
 
   test('should return false if settled and has no matched payment request', async () => {
```

Comment:
Should the name of this test not be changed to be 'return undefined'?

Reply:
This part is the level above the update module.  This one still returns true or false and is unchanged so all good :)

Reply:
Aah I didn't see rest of the test - yeah that's fine